driver/config/config.go

```diff
@@ -146,7 +147,11 @@ type (
 var Argon2DefaultParallelism = uint8(runtime.NumCPU() * 2)
 
 func HookStrategyKey(key, strategy string) string {
-	return fmt.Sprintf("%s.%s.hooks", key, strategy)
+	if strategy == "none" {
```

Comment:
Nitpick: This took me a while to track. It may be only me but probably more useful name for this could be something like `global` or `all`? I get the intention but for me it was a bit misleading.

Reply:
I see this is being discussed already 😉

Reply:
ok. Will change it back to global ;)